torch/_tensor.py

```diff
@@ -1026,6 +1027,61 @@ def __torch_function__(cls, func, types, args=(), kwargs=None):
             else:
                 return _convert(ret, cls)
 
+    def __dlpack__(self, stream=None):
+        """
+        Creates a dlpack capsule of the current tensor to
```

Comment:
"Creates a DLPack capsule from the current tensor."

The phrase "DLPack capsule" should be a link to the DLPack documentation.